doc/INSTALL.md

```diff
@@ -41,12 +41,14 @@ Get dependencies:
       gettext
 
 If you don't have Bitcoin installed locally you'll need to install that
-as well:
-
-    sudo apt-get install software-properties-common
-    sudo add-apt-repository ppa:bitcoin/bitcoin
-    sudo apt-get update
-    sudo apt-get install -y bitcoind
+as well. It's now available via [snapd](https://snapcraft.io/bitcoin-core).
+
+    sudo apt-get install snapd
+    sudo snap install bitcoin-core
+    # Snap does some weird things with binary names; you'll
+    # want to add a link to them so everything works as expected
+    sudo ln -s /snap/bitcoin-core/current/bin/bitcoind /usr/bin/bitcoind
+    sudo ln -s /snap/bitcoin-core/current/bin/bitcoin-cli /usr/bin/bitcoin-cli
```

Comment:
Probably want /usr/local/bin?

`sudo ln -s /snap/bitcoin-core/current/bin/bitcoin{d,-cli} /usr/local/bin/`?

Reply:
updated, but i seem to remember @rustyrussell mentioning a problem with the `ln` remap on running a node?